mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/ElasticSearchIntegrationTest.java

```diff
@@ -35,7 +35,7 @@
 import org.apache.james.mailbox.inmemory.InMemoryMailboxSessionMapperFactory;
```

Comment:
Why? Oh I see... I changed the name in the same time, but as you can see below (its not very clear in github) the Authenticator has been moved. That's the essential point of this commit.